components/suggest/src/store.rs

```diff
@@ -3883,8 +3916,8 @@ pub(crate) mod tests {
         );
 
         // Dismiss the first suggestion.
-        assert_eq!(suggestions[0].dismissal_key(), Some("dk0"));
-        store.inner.dismiss_by_suggestion(&suggestions[0])?;
+        assert_eq!(suggestions_0[0].dismissal_key(), Some("dk0"));
+        store.inner.dismiss_by_suggestion(&suggestions_0[0])?;
```

Comment:
Could you assert the following here please?

* `store.inner.is_dismissed_by_suggestion` returns true for this suggestion
* `store.inner.any_dismissed_suggestions` returns true